integration-cli/requirements_test.go

```diff
@@ -69,6 +69,10 @@ func UnixCli() bool {
 	return isUnixCli
 }
 
+func NotGitHubActions() bool {
+	return os.Getenv("GITHUB_ACTIONS") != "true"
```

Comment:
Better to use strconv.ParaeBool

Reply:
Yes, I think we can _either_ go with "allow the variable to both enable _or_ explicitly _disable_", in which case we would have something like [`gzDecompress()`](https://github.com/moby/moby/blob/0910306bf970603ce787466a98e4294ba81af841/pkg/archive/archive.go#L194-L200)

Or, we could keep it simple and consider any non-empty value to be "enabled", and empty to be "disabled"; https://github.com/moby/moby/blob/9184f0b5e40cd94fdb96d8fe42f74f028aa17b9a/distribution/push_v2.go#L198

Given that this is only for CI, and not a user-facing option, I'd be ok with "non-empty" means "enabled";

```suggestion
	return os.Getenv("GITHUB_ACTIONS") == ""
```

Or is it expected for this var to have `GITHUB_ACTIONS=false` ? I see various project just checking for "empty", but there's some that have "false" conditions as well; https://grep.app/search?q=GITHUB_ACTIONS&case=true&filter[lang][0]=Shell

Reply:
Non empty sgtm